.circleci/config.yml

```diff
@@ -69,7 +69,7 @@ jobs:
           command: |
             set -euo pipefail
             export CELO_MONOREPO_DIR="$PWD"
-            git clone --depth 1 https://github.com/celo-org/celo-monorepo.git ${CELO_MONOREPO_DIR} -b master
+            git clone --depth 1 https://github.com/celo-org/celo-monorepo.git ${CELO_MONOREPO_DIR} -b bricewang/registry-lookup
```

Comment:
Same comment as on fixidity, you will only need to repoint the end-to-end tests for the first PR, and then the second PR can point to master